core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -240,6 +298,23 @@ private boolean isSelectedSubnetAvailableAndRequestedType(Network network, Envir
         return false;
     }
 
+    private String getSubnetId(Network network) {
+        Json attributes = network.getAttributes();
+        Map<String, Object> params = attributes == null ?
+                Collections.emptyMap() : attributes.getMap();
+
+        return params.get(SUBNET_ID) != null ? String.valueOf(params.get(SUBNET_ID)) : null;
```

Comment:
could you use the constant from `NetworkConstants` instead of redefining it? thanks
also please be sure to delete unused methods and variables